### ISSUE TITLE
Inconsistent/incorrect output when trying to translate a paragraph

### ISSUE THREAD

**Issue by NehaPathapati** [tensorflow/tensor2tensor#867]
### Description
I am using the checkpoints available for the English to German translation task. 

The problem is translate_ende_wmt32k. 
The model is transformer.
The hyper-parameters set used is transformer_base.

These are the translations got from the model:
EX1:
ENGLISH: Today is a hot day in Folsom. The temperature is about a 30 degree Celsius.
GERMAN: Heute ist ein heißer Tag in Folsom, die Temperatur beträgt etwa 30 Grad Celsius.

EX2:
ENGLISH: Today is a hot day in Folsom. The temperature is about a 30 degree Celsius. Don't forget your sunglasses.
GERMAN: Die Temperaturen liegen bei 30 Grad Celsius und vergessen Sie nicht Ihre Sonnenbrille.

EX3:
ENGLISH: Today is a hot day in Folsom. The temperature is about a 30 degree celsius. Don't forget your sunglasses and sunscreen. Also, make sure you drink lots of water.
GERMAN: Die Temperaturen liegen bei 30 Grad Celsius und die Sonnenbrille und die Sonnencreme sind ebenfalls sehr angenehm.

These were the observations I made:
The German translation will never have multiple sentences, even if the input English text had. The German translation is always comma separated.
When the English text has multiple sentences, it is not translating all of them.

Is anyone else getting similar outputs? Am I doing something wrong to get such outputs? 

Does the model always behave this way?

Any help will be greatly appreciated.

### Environment information

 
```
OS: <your answer here>

$ pip freeze | grep tensor
# your output here

$ python -V
# your output here
```

### For bugs: reproduction and error logs

```
# Steps to reproduce:
...
```

```
# Error logs:
...
```

**Comment by martinpopel** [comment]
This is the expected behavior because in the en-de training data most segments are actually single sentences, so the system learns that the target segment almost never contains two or more sentences.
You should segment the test data one sentence per line before running `t2t-decoder`.

Alternatively, you may try editing the training data and putting two or three sentences per line. This could have the advantage that the system learns translating cross-sentence phenomena (e.g. coreference), but it has several disadvantages/caveats: e.g. longer training segments result in less segments per batch and this can influence the training convergence. You should mix in also some single-sentence segments into the training data, so that you can translate also single sentences if needed.

In general: the best results are achieved if the test data resembles the training data. And sentence length is one of the important properties.
